contents/tutorials/aa-testing.md

```diff
@@ -0,0 +1,92 @@
+
+Teams run A/A tests to ensure their A/B test service, functionality, and implementation process results in accurate and relevant results. A/A tests are a way to discover broken parts of this process, and this tutorial shows you how to run one in PostHog.
+
+> Unlike other A/B testing tools, A/A tests aren’t needed to calculate baseline conversion and sample metrics in PostHog. You can calculate those using your product data, and they are automatically when you create an experiment.
+
+## Creating an experiment
+
+The first step in running an A/A test is creating an experiment. You do this going to the [experiments tab](https://app.posthog.com/experiments) and clicking the "New experiment" button. Set a name, key, and experiment goal. Make sure to clarify in the name or description that it is an A/A test.
+
+![Create an experiment](../images/tutorials/aa-testing/experiment.png)
+
+If you use an absolute goal, like total pageviews, make sure to use an equal variant split (which is default). If you use a relative goal like conversion, you can test non-equal variant splits like 25-75. To do this, edit the feature flag with the same key as your experiment, change the variant rollout, and press save.
+
+## Implementing your A/A test
+
+Critical to A/A testing is treating it like a standard A/B test. This means implementing it like any A/B test. You must call the related feature flag, set up code dependent on it, and capture the relevant goal event(s).
+
+<MultiLanguage>
+
+```js
+'use client'
+import { useFeatureFlagVariantKey } from 'posthog-js/react'
+import Link from 'next/link'
+
+export default function Home() {
+
+  const test = useFeatureFlagVariantKey('aa-homepage') === 'test'
+  const control = useFeatureFlagVariantKey('aa-homepage') === 'control'
+
+  return (
+    <>
+      <h1>Welcome to the homepage</h1>
+      <h2>{ test || control ? <Link href='/signup'>Sign up</Link> : ''}</h2>
+    </>
+  )
+}
+```
+
+```python
+from posthog import Posthog
+
+posthog = Posthog(
+  '<ph_project_api_key>', 
+  host='<ph_instance_address>'
+)
+user_id = 'ian@posthog.com'
+
+flag = posthog.get_feature_flag('aa-homepage', user_id)
+
+if flag == 'test' or flag == 'control':
+    posthog.capture(user_id, 'ran query', {'$feature/aa-homepage': flag})
+    # do something...
+```
+
+</MultiLanguage>
+
+After implementing, go back to your [experiment](https://app.posthog.com/experiments) and click "launch." Let the experiment run for a few days or weeks and monitor its metrics.
+
+## Handling A/A test results
+
+After running your tests, you should see the same results for all the variants, or if not the same, the results should not be statistically significant. The probability a variant is the best should be about 50% for both.
```

Comment:
```suggestion
After running your tests, you should see the similar results for all the variants. The results should not be statistically significant, since all variants received the same treatment.
```